packages/ui-extensions-react/src/surfaces/admin/render.tsx

```diff
@@ -17,7 +17,10 @@ import {ExtensionApiContext} from './context';
  * hook.
  *
  * @param target The extension point you are registering for. This extension
- * point must be a `RenderExtensionPoint`;
+ * point must be a `RenderExtensionPoint`; if you are trying to register for a non-
+ * rendering extension point, like `Admin::Customers::SegmentationTemplatesFirstTimeBuyers::Show`, use the
+ * `extension()` function provided by this library instead.
```

Comment:
Similar to what checkout is doing with `Checkout::PostPurchase::ShouldRender`. As a consumer, am I expected to use `extension`?:

```
import { extension } from '@shopify/ui-extensions-react/admin';

extension('Admin::Customers::SegmentationTemplatesFirstTimeBuyers::Show',  ({i18n, enabledFeatures}) => {
 return [ list of templates ]
})
```

`extension` doesn't inject the `ExtensionApiContext`. Should pass our API as param instead of reading from the context?